readme.md

```diff
@@ -65,6 +65,12 @@ if (window.plugins.touchid) {
 Call the function you like
 
 **isAvailable(successCallback, errorCallback(msg))** will Check if touchid is available on the used device 	
+
+**biometricType(successCallback(msg), errorCallback(msg))** 
+assumes that isAvailabe has been called
+returns FACE, TOUCH, NONE for biometric type
+currently implemented for ios.  android api currently returns NONE. 
```

Comment:
sorry i didn't see this before, but the android API now returns NONE or TOUCH, correct?